about.html

```diff
@@ -0,0 +1,240 @@
+                            <div class="past_summit_img">
+                                <img src="./assets/images/past_summit.jpg" alt="" srcset="">
+                            </div>
+                            <div class="past_summit_info_wrapper">
+                                <div class="past_summit_info">
+                                    <p class="past_summit_date">2013</p>
+                                    <p class="past_summit_location">CC Global Summit 2013 in Warsaw</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="past_summit_img_wrapper">
+                        <div class="past_summit_img_container">
+                            <div class="past_summit_img">
+                                <img src="./assets/images/past_summit.jpg" alt="" srcset="">
+                            </div>
+                            <div class="past_summit_info_wrapper">
+                                <div class="past_summit_info">
+                                    <p class="past_summit_date">2013</p>
+                                    <p class="past_summit_location">CC Global Summit 2013 in Warsaw</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="partner_wrapper hide--desktop">
+        <div class="partner">
+            <div class="partner_title_wrapper">
+                <div class="partner_title_container">
+                    <h2 class="partner_title">Partner</h2>
+                    <hr class="partner_bar">
+                </div>
+            </div>
+
+            <div class="partner_logo_wrapper">
+                <div class="partner_logo">
+                    <img src="./assets/images/partner1.png" alt="partner logo">
+                    <img src="./assets/images/partner2.png" alt="partner logo">
+                    <img src="./assets/images/partner3.png" alt="partner logo">
+                    <img src="./assets/images/partner4.png" alt="partner logo">
+                    <img src="./assets/images/partner5.png" alt="partner logo">
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="footer_wrapper">
+        <footer>
+            <div class="footer_container">
+                <div class="footer_img">
+                    <img class="hide--desktop" src="./assets//images/footer_logo.png" alt="logo footer">
+                    <img class="hide--mobile logo_footer_desktop" src="./assets//images/footer_about.png"
+                        alt="logo footer">
+                </div>
+                <div class="footer_rights">
+                    <p>
+                        2015 Creative Commons Korea. Some Rights Reserved.
```

Comment:
- [ ] It could be better if you use the 👉🏽 current year 👈🏽  instead of 2015 and the 👉🏽 name of your page 👈🏽 instead of "Creative Commons" to personalize your page.